index.d.ts

```diff
@@ -20,16 +21,16 @@ declare namespace dashjs {
     }
 
     export interface MediaInfo {
-        id: number | null;
+        id: string | null;
```

Comment:
Hmm... looking back at this, I'm not 100% sure that `string` is the correct type. From looking at the source code, I thought `mediaInfo.id` would be set to a string. But it's possible that I misread the code.